recipes/r-redlistr/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set version = '1.0.3' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-redlistr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/redlistr_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/redlistr/redlistr_{{ version }}.tar.gz
+  sha256: 9949f23e668e7b560590993d3e0aa6502fb96fed6a67bbcfe13ebf20abeb7200
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-plyr
+    - r-raster >=2.5_8
+    - r-rgeos
+    - r-sp >=1.2_4
+  run:
+    - r-base
+    - r-plyr
+    - r-raster >=2.5_8
+    - r-rgeos
+    - r-sp >=1.2_4
+
+test:
+  commands:
+    - $R -e "library('redlistr')"           # [not win]
+    - "\"%R%\" -e \"library('redlistr')\""  # [win]
+
+about:
+  home: https://github.com/red-list-ecosystem/redlistr
+  license: GPL-3
+  summary: A toolbox created by members of the International Union for Conservation of Nature
+    (IUCN) Red List of Ecosystems Committee for Scientific Standards. Primarily, it
+    is a set of tools suitable for calculating the metrics required for making assessments
+    of species and ecosystems against the IUCN Red List of Threatened Species and the
+    IUCN Red List of Ecosystems categories and criteria. See the IUCN website for detailed
+    guidelines, the criteria, publications and other information.
+  license_family: GPL3
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
```

Comment:
Authors also have a separate [`LICENSE` file](https://github.com/red-list-ecosystem/redlistr/blob/master/LICENSE) in the package, which should also be included. The DESCRIPTION file shows `License: GPL (>= 3) | file LICENSE`, indicating the authors are intent on its inclusion.

Reply:
@Marie59 still need to add the LICENSE file to the list here.